### PR TITLE
OEL-594: [oe_showcase] views

### PR DESCRIPTION
**Pull request by abel-santos-corral**
### Description

View for news.

### CODE REVIEW

File: modules/oe_showcase_default_content/oe_showcase_default_content.info.yml [hunk 1]
```diff
@@ -12,6 +12,42 @@ dependencies:
   - oe_showcase_page:oe_showcase_page
 
 default_content:
+  file:
+    # OE Showcase Search
```
Comment (drishu) [inline modules/oe_showcase_default_content/oe_showcase_default_content.info.yml:16]: All these that are not News related seem out of place, either rebase the epic or drop the commits that adds this (if you add a new commit that removes this then you will have a conflict when trying to merge or worse, the epic PR will try to remove these)

File: modules/oe_showcase_default_content/oe_showcase_default_content.info.yml [hunk 2]
```diff
@@ -44,49 +80,25 @@ default_content:
     - a170562d-55f5-48b5-9e11-7a61d7677873
     - e0bff251-495d-4dbe-b723-62e774ef6238
     - 073cc92d-359d-4195-bf9e-6d49459d1f86
+    # OE Showcase News
+    - 0708ba1e-f163-4758-a201-e5c087079b1b
```
Comment (drishu) [inline modules/oe_showcase_default_content/oe_showcase_default_content.info.yml:84]: to many, we already demo the pagging in the search page, lets add just 5 items, all with same image

File: modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+region: sidebar_left
+weight: 0
+provider: null
+plugin: 'facets_form:search_api:views_page__showcase_search__showcase_search_page'
+settings:
+  id: 'facets_form:search_api:views_page__showcase_search__showcase_search_page'
+  label: 'Filter Options'
+  provider: facets_form
+  label_display: visible
+  live_total: true
```
Comment (drishu) [inline modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml]: false

File: modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  theme:
+    - oe_whitelabel
+id: showcase_facets_form
+theme: oe_whitelabel
+region: sidebar_left
+weight: 0
+provider: null
+plugin: 'facets_form:search_api:views_page__showcase_search__showcase_search_page'
+settings:
+  id: 'facets_form:search_api:views_page__showcase_search__showcase_search_page'
```
Comment (drishu) [inline modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml]: the id seems odd, shouldn't this be for the news view ?

File: modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  label_display: visible
+  live_total: true
+  button:
+    label:
+      submit: Refine
+      reset: Clear
+  facets: {  }
+visibility:
+  request_path:
+    id: request_path
```
Comment (drishu) [inline modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml]: I think the entire file has no place here

File: modules/oe_showcase_news/config/install/core.entity_view_display.node.oe_news_simple.search_result.yml [hunk 6]
```diff
@@ -19,7 +19,7 @@ bundle: oe_news_simple
 mode: search_result
 content:
   field_news_content:
-    type: text_summary_or_trimmed
+    type: text_trimmed
```
Comment (drishu) [inline modules/oe_showcase_news/config/install/core.entity_view_display.node.oe_news_simple.search_result.yml]: why don't we like the summary ?

File: modules/oe_showcase_search/config/install/block.block.showcase_facets_form.yml [hunk 7]
```diff
@@ -26,6 +26,7 @@ settings:
       submit: Refine
       reset: Clear
   facets: {  }
+  live_total: true
```
Comment (drishu) [inline modules/oe_showcase_search/config/install/block.block.showcase_facets_form.yml]: false

File: modules/oe_showcase_news/oe_showcase_news.info.yml [hunk 8]
```diff
@@ -28,6 +28,15 @@ config_devel:
...
+    - pathauto.pattern.news_url_alias_pattern
+    - block.block.page_title_news
+    - search_api.server.showcase_news_server
+    - search_api.index.showcase_news_index
+    - views.view.oe_news
+    - search_api_autocomplete.search.oe_news
+    - facets.facet_source.search_api__views_page__oe_news__showcase_news_page
+    - facets.facet.publication_date
+    - block.block.showcase_news_facets_form
+    - block.block.showcase_facets_form
```
Comment (drishu) [inline modules/oe_showcase_news/oe_showcase_news.info.yml]: you should remove this

File: modules/oe_showcase_news/templates/node--oe-news-simple.html.twig [hunk 9]
```diff
@@ -0,0 +1,36 @@
+{#
+/**
+ * @file
+ * Search result template.
+ */
+#}
+{% set badges = [] %}
```
Comment (drishu) [inline modules/oe_showcase_news/templates/node--oe-news-simple.html.twig]: this template is outdated, update the news epic, and copy what was done in the search template

File: modules/oe_showcase_news/templates/node--oe-news-simple.html.twig [hunk 10]
```diff
@@ -0,0 +1,36 @@
+{#
```
Comment (drishu) [inline modules/oe_showcase_news/templates/node--oe-news-simple.html.twig]: the template suggestion used is false, this is applying also the the node view, it should only apply to search results

**Review by drishu — CHANGES_REQUESTED** [review]

File: modules/oe_showcase_default_content/oe_showcase_default_content.info.yml (hunk 2 above)
Comment (GilNovacomm) [reply modules/oe_showcase_default_content/oe_showcase_default_content.info.yml:84]: These content related changes were done on OEL-596. I've updated this branch and removed the excess default content. 594 will be rebased to contain these changes.

File: modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml (hunk 3 above)
Comment (GilNovacomm) [reply modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml]: This file will be removed it is not part of news.

File: modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml (hunk 4 above)
Comment (GilNovacomm) [reply modules/oe_showcase_news/config/install/block.block.showcase_facets_form.yml]: This file should not have been added to news. I'll remove it

File: modules/oe_showcase_news/oe_showcase_news.info.yml (hunk 8 above)
Comment (GilNovacomm) [reply modules/oe_showcase_news/oe_showcase_news.info.yml]: removed: block.block.showcase_facets_form

File: modules/oe_showcase_news/templates/node--oe-news-simple.html.twig (hunk 10 above)
Comment (GilNovacomm) [reply modules/oe_showcase_news/templates/node--oe-news-simple.html.twig]: Agreed updated it to be used only with search results.

**Comment by brummbar** [comment]
Closing as outdated.

**Pull request closed without merging**